aexpect/client.py

```diff
@@ -166,7 +167,12 @@ def __init__(self, command=None, a_id=None, auto_close=False, echo=False,
 
         # Start the server (which runs the command)
         if command:
-            helper_cmd = utils_path.find_command('aexpect-helper')
+            # try to find python specific version of aexpect-helper first, then try unversioned
+            helper_noversion = 'aexpect-helper'
+            helper_versioned = helper_noversion + '-{0.major}.{0.minor}'.format(sys.version_info)
+            helper_cmd = utils_path.find_command(cmd_name_versioned, "NOTFOUND")
```

Comment:
You probably meant:

```python
helper_cmd = utils_path.find_command(helper_versioned, "NOTFOUND")
```

So the obvious fix is `s/cmd_name_versioned/helper_versioned/`, but I suggest you use:

```python
            try:
                helper_cmd = utils_path.find_command(helper_versioned)
            except utils_path.CmdNotFoundError:
                helper_cmd = utils_path.find_command(helper_noversion)
```

Reply:
Good heavens. Yes. How did I accidentally check in that "work in progress" line? Updated as suggested. Thanks!